src/RageThreads.cpp

```diff
@@ -682,11 +682,17 @@ void LockMutex::Unlock()
 
 	mutex.Unlock();
 
-	if( file && locked_at != -1 )
+	// FAST_ULL_MAX is used here to signify an invalid time, since it's equivalent
```

Comment:
Change the constructor of LockMutex to use FAST_ULL_MAX instead of -1. We don't need this comment then.

Reply:
I think we already are...?
 
Line 139 of RageThreads.h: `LockMutex(RageMutex &mut): mutex(mut), file(nullptr), line(-1), locked_at(FAST_ULL_MAX), locked(true) { mutex.Lock(); }`

I will get rid of that comment though

Reply:
Oh yeah woops I was looking at beta and not this PR so yeah we're good!